.github/workflows/release.yaml

```diff
@@ -9,12 +9,35 @@ jobs:
   release:
     runs-on: ubuntu-latest
     timeout-minutes: 60
+    strategy:
+      matrix:
+        node-version: ['lts/*']
+      fail-fast: false
     steps:
-      - name: Checkout
+      - name: Checkout project
         uses: actions/checkout@v3
 
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      - name: Cache Node dependencies
+        uses: actions/cache@v3
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-
+
+      - name: npm install & build
+        run: |
+          npm install
+          npm run build --if-present
```

Comment:
This will run the build script, which in turn produces the binaries in the `dist` directory. Do we want to update the `node.yaml` workflow that runs on pull requests to upload the dist files as an artifact, so we have a way to download those files from a PR?

Reply:
I don't think it's necessary to have artifacts for PRs but I'm not against it either.

Reply:
Sounds good! I agree...we can always add them later if we find we need them.